code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -305,5 +305,13 @@
 		/obj/item/warp_core,
 		/obj/item/weapon/extraction_pack,
 		/obj/item/weapon/rrf,
-		/obj/item/weapon/gun/custom_ka/
+		/obj/item/weapon/gun/custom_ka
 		)
+
+/obj/item/weapon/storage/belt/bandolier
+	name = "bandolier"
+	desc = "A pocketed belt designated to hold shotgun shells."
+	icon_state = "bandolier"
+	item_state = "bandolier"
+	can_hold = list(/obj/item/ammo_casing/shotgun)
+	storage_slots = 16
```

Comment:
How does this look when opening it via menu? It may take up a lot of space because each storage space is a ~20x20 box that takes up the screen. It might be better to just use what boxes do and set `max_storage_space` to an appropriate value and `storage_slots` to null.